### PR TITLE
Hue sensors

### PR DESCRIPTION
**Pull request by robmarkcole**
**Description:**
Adds docs fur hue_sensors component

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant/pull/9796):** home-assistant/home-assistant#<9796>

### CODE REVIEW

File: source/_components/sensor.hue.markdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+logo: philips_hue.png
+ha_category: Sensor
+ha_iot_class: "Cloud Polling"
+ha_release: 0.55
+---
+
+
+The `hue` sensor will display the state of Hue [SML001](https://www.philips.co.uk/c-p/8718696595190/hue-motion-sensor) motion sensors and [RWL021](http://www2.meethue.com/en-us/p/hue-dimmer-switch/046677458140) remotes (dimmer switch). This sensor scans through a list of all sensors on your system and adds a single entity for each SML001 or RWL021 discovered.
+
+The RWL021 remote has a total of 8 possible state, corresponding to the remote having 4 buttons, where each button can be short pressed (clicked) and long pressed (when holding the button for 2 sec you will see the LED blink twice).
```
Comment (frenck) [inline source/_components/sensor.hue.markdown]: **Spelling & Grammar**:
- `state` -> `states`

File: source/_components/sensor.hue.markdown [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+The SML001 detects motion using a PIR, but also has sensors for temperature and light level. The state of the sensor indicates whether motion is detected. whilst temperature and light level data is recorded in the sensor attributes.
```
Comment (frenck) [inline source/_components/sensor.hue.markdown]: **Spelling & Grammar**:
- Unnecessary comma after `PIR`
- Determiner before `motion` missing -> `whether a motion is detected`
- `whilst` -> `Whilst`
- `data is recorded` -> `data are recorded`

**Comment by robmarkcole** [comment]
Made requested changes, thanks @frenck

File: source/_components/sensor.hue.markdown [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+logo: philips_hue.png
+ha_category: Sensor
+ha_iot_class: "Cloud Polling"
+ha_release: 0.55
+---
+
+
+The `hue` sensor will display the states of Hue sensors registered to your Hue hub. The component supports [SML](https://www.philips.co.uk/c-p/8718696595190/hue-motion-sensor) motion sensors, [RWL](http://www2.meethue.com/en-us/p/hue-dimmer-switch/046677458140) dimmer and [ZPG](https://www.philips.co.uk/c-p/8718696498026/hue-tap-switch) TAP remotes. Additionally if you have configured a Geofence in the Hue app, the state will be reported. The properties of these sensors are described in detail on the [Hue Developers website](https://developers.meethue.com/documentation/supported-sensors).
+
+The RWL dimmer remote has a total of 8 possible state, corresponding to the remote having 4 buttons, where each button can be short pressed (clicked) and long pressed (when holding the button for 2 sec you will see the LED blink twice). The ZPG (or Tap) remote has 4 buttons.
```
Comment (frenck) [inline source/_components/sensor.hue.markdown]: The following error is still in place:
- `state`  -> `states`

**Comment by robmarkcole** [comment]
Perfect, cheers!

**Comment by frenck** [comment]
Closing PR since parent PR has been closed as well. Feel free to reopen this PR when needed.

**Pull request closed without merging**